Orange/canvas/application/errorreporting.py

```diff
@@ -184,12 +188,15 @@ def _find_last_frame(tb):
                 tb = tb.tb_next
             return tb
 
-        err_locals, err_module, frame = None, None, _find_last_frame(tb)
+        err_module, frame = None, _find_last_frame(tb)
```

Comment:
Leave `err_locals=None` default even when `not frame`? Then you can `pass` the exception below.

Reply:
Yes, firstly I did so. Reverting back.

Reply:
I think it's safer since `err_locals` is used _unconditionally_ further below.